src/utils/aggregateTypeErrors.ts

```diff
@@ -0,0 +1,15 @@
+const unsupportedAggregateType = 'unsupported aggregate column type'
+
+export const isAggregateTypeError = (
+  errorCode: string,
+  errorMessage: string
+): boolean =>
+  errorCode === 'invalid' && errorMessage.includes(unsupportedAggregateType)
+
+export const aggregateTypeError = () => {
+  return {
+    aggregateType: true,
+    message:
+      "It looks like you're trying to apply an incompatible aggregate function. You can fix this by selecting the Last Aggregate Function",
+  }
```

Comment:
Could we explain "incompatible" a bit more? Something like `It looks like you're trying to apply a number-based aggregate function to a string, which cannot be processed. You can fix this by selecting the Aggregate Function "Last"`

Reply:
Sounds great!